scheduler/config.go

```diff
@@ -36,6 +39,23 @@ type Config struct {
 	WorkManagerPoolSize  uint `json:"work_manager_pool_size,omitempty"yaml:"work_manager_pool_size,omitempty"`
 }
 
+const (
+	CONFIG_CONSTRAINTS = `
+	"scheduler": {
+					"type": ["object", "null"],
+					"properties" : {
+									"work_manager_queue_size" : {
+													"type": "integer"
+									},
+									"work_manager_pool_size" : {
+													"type": "integer"
+									}
```

Comment:
We should add minimum value of 1 for all these so we can error on 0/negative values.